.github/workflows/build.yaml

```diff
@@ -24,6 +22,9 @@ on:
         required: false
         type: string
         default: yarn
+      monorepo_manager:
```

Comment:
Need to fix this. We should check which monorepo versions we should support. 
Right now only `yarn workspaces` works/is tested. I guess that could be good enough for now